src/briefcase/commands/base.py

```diff
@@ -957,20 +966,22 @@ def _generate_template(self, template, branch, output_path, extra_context):
         # Make sure we have an updated cookiecutter template,
         # checked out to the right branch
         cached_template = self.update_cookiecutter_cache(
-            template=template, branch=branch
+            template=template,
+            branch=branch,
         )
 
         self.logger.configure_stdlib_logging("cookiecutter")
 
         try:
             # Unroll the template. Use a copy of the context to ensure that any
-            # mocked calls have an unmodified copy.
+            # mocked calls have an unmodified copy. Disable the use of replay.
             self.tools.cookiecutter(
                 str(cached_template),
                 no_input=True,
                 output_dir=str(output_path),
                 checkout=branch,
                 extra_context=extra_context.copy(),
+                replay=False,
```

Comment:
I'll still need to look more closely at the repo mgmt....but I can say that this will not disable capturing replay data, unfortunately. The `replay` argument only has affect when truthy and that's to perform a "replay"....the replay data is still captured in `~/.cookiecutter_replay`, irrespective.

Reply:
Sigh... oh well, a full configuration file it is then.

Reply:
That definitely sounds preferable; better still would be passing in the parsed configuration as a dict (so we can pass in an explicit replay path without needing to "fake" a YAML file).

On that basis, I'll leave this as is for now (disabling replay is still an improvement), and update the release note to only talk about the `.cookiecutters` folder.

Reply:
- https://github.com/beeware/briefcase/issues/1857